gui/epics_signal.py

```diff
@@ -0,0 +1,40 @@
+from qtpy.QtCore import Signal, QObject
+from epics import PV
+
+
+class EpicsQObject(QObject):
+    """
+    Class to simplify the process of triggering a qt signal based on an epics PV
+    Previously there were multiple unique signals defined and their corresponding callbacks
+    But their code is not unique. This class attempts to reduce the noise
+    """
+
+    # Define the PyQt signal
+    epics_pv_changed = Signal(object)
+
+    def __init__(self, pv_name, qt_callback, use_string=False, pv_callback=None, custom_pv=None):
+        super().__init__()
+        self.use_string = use_string
+        # Define the pyepics PV and its callback
+        if not pv_callback:
+            pv_callback = self.on_pv_changed
+
+        if custom_pv:
```

Comment:
There are two things strange here:
* Is this a PV object or a function? it looks like a function because `self.pv` calls it like that
* I see no instance of `EpicsQObject` that uses this kwarg or even as a positional arg - is this necessary?